File: dev/20190216/k2works/FizzBuzz.py
import unittest

class TestFizzBuzz(unittest.TestCase):
  def testGreeting(self):
    self.assertEqual('Hello, World!', FizzBuzz.greeting)

class FizzBuzz:
  def greeting(self):
    return 'Hello, World!'

if __name__ == "__main__":
  unittest.main()

File: dev/20190216/k2works/FizzBuzz.exs
ExUnit.start()

defmodule FizzBuzzTest do
  use ExUnit.Case, async: true

  test "Hello world" do
    assert FizzBuzz.hello() == "Hello world"
  end
end

defmodule FizzBuzz do
  def hello do
    "Hello world"
  end
end



